enclave/cert.c

```diff
@@ -532,6 +567,9 @@ static int _parse_extensions(
     ret = 0;
 
 done:
+    if (ret != 0)
+        OE_TRACE_ERROR("rc=0x%x", rc);
```

Comment:
Since you are invoking OE_TRACE_ERROR at the done label, you can remove the instances that you added earlier as they become redundant.

Reply:
addressed